Interop/Runtime/src/native/kotlin/kotlinx/cinterop/NativeMem.kt

```diff
@@ -118,14 +118,22 @@ internal object nativeMemUtils {
     }
 
     fun alloc(size: Long, align: Int): NativePointed {
+        return interpretOpaquePointed(allocRaw(size, align))
+    }
+
+    fun free(mem: NativePtr) {
+        freeRaw(mem)
+    }
+
+    fun allocRaw(size: Long, align: Int): NativePtr {
         val ptr = malloc(size, align)
         if (ptr == nativeNullPtr) {
             throw OutOfMemoryError("unable to allocate native memory")
         }
-        return interpretOpaquePointed(ptr)
+        return ptr
     }
 
-    fun free(mem: NativePtr) {
+    fun freeRaw(mem: NativePtr) {
```

Comment:
`private`?

Reply:
I'll make both `allocRaw` and `freeRaw` internal for uniformity.